### PR TITLE
add the equalizer

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
## Summary
add the equalizer to the project

##Screenshot
captura de pantalla 2018-09-06 a la s 10 50 54

## Trello Card
https://trello.com/c/0LaCZL8q/19-css-resets

### CODE REVIEW

File: scss/styles.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.title {
+    color: red;
+  }
```
Comment (mmamaniwolox) [inline scss/styles.scss]: Delete tabulation before curly brace `}`

File: scss/styles.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.title {
+    color: red;
```
Comment (mmamaniwolox) [inline scss/styles.scss]: We use tabulation of 2 spaces, here you have 4

File: scss/styles.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+  }
+
```
Comment (mmamaniwolox) [inline scss/styles.scss]: Add a tailing line

File: scss/application.scss [hunk 4]
```diff
@@ -1,3 +1,7 @@
+@import 'equalizer';
+@import 'styles';
+
 .title {
   color: red;
 }
+
```
Comment (mmamaniwolox) [inline scss/application.scss]: I think you have 2 tailing lines here? Use just one

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: scss/application.scss [hunk 5]
```diff
@@ -1,3 +1,6 @@
+@import 'equalizer';
+@import 'styles';
+
 .title {
```
Comment (mmamaniwolox) [inline scss/application.scss]: This is now imported from `@import 'styles';`.
Delete this style from this file

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 6]
```diff
@@ -1,3 +1,8 @@
 .sass-cache/
 application.css
 application.css.map
+styles.css
+styles.css.map
+equalizer.css
+equalizer.css.map
```
Comment (mmamaniwolox) [inline .gitignore]: I think you only need `application` ones here. Remove both `styles.css and styles.css.map` and the ones for equalizer

File: .gitignore [hunk 7]
```diff
@@ -1,3 +1,8 @@
...
+
```
Comment (FrankIglesias) [inline .gitignore]: Delete this line

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 7 above)
Comment (FrankIglesias) [reply .gitignore]: Add something like *.css and *.css.map

**Review by FrankIglesias — APPROVED** [review]

**Review by mmamaniwolox — APPROVED** [review]

**Pull request merged**